### ISSUE TITLE
Strange method implementation for MathHelper.LerpPrecise

### ISSUE THREAD

**Issue by ala53** [mono/MonoGame#4279]
In MonoGame.Framework/MathHelper.cs, line 196:
```
        public static object LerpPrecise(float value1, float value2, float amount)
        {
            return ((1 - amount) * value1) + (value2 * amount);
        }
```

Is there a reason this returns an object (which is both useless to API users and involves boxing/GC overhead) rather than a float?

**Comment by KonajuGames** [comment]
It shouldn't. We missed that when reviewing the PR. We concentrated on what
it did compared to XNA rather than how it returned it.

**Comment by ala53** [comment]
Oh, well that makes things easy. Should I do a PR?

**Comment by ala53** [comment]
To be fixed in #4280
